core/themes/seven/css/vertical-tabs.css

```diff
@@ -7,7 +7,10 @@
   border: 1px solid #ccc;
   background: #fff;
 }
- @media (min-width: 750px) {
+[dir="rtl"] .vertical-tabs:before {
+  right: 0;
```

Comment:
Hm, where is the corresponding LTR rule for this? I suppose it's implied just because left: 0 is the default? I think we need to clarify this by adding:

```
.vertical-tabs:before {
  left: 0; /* LTR */
}
```

Reply:
There already was one, further down, within the `@media (min-width: 750px)` ...moved things around.